lib/logdna.rb

```diff
@@ -82,78 +59,69 @@ def level=(value)
       @level = value
     end
 
-    def log(msg=nil, opts={})
-      loggerExist?
-      message = msg
-      message = yield if msg.nil? && block_given?
-      @response = @@client.buffer(message, default_opts.merge(opts).merge({
-            timestamp: (Time.now.to_f * 1000).to_i
-        }))
-      'Saved'
+    def log(message = nil, opts = {})
+      if message.nil? && block_given?
+        message = yield
+      elsif message.nil?
```

Comment:
Might want to make this just `if message.nil?` to detect if the block provided is wrong.

Reply:
good point